build/templates/errors.py.mako

```diff
@@ -55,6 +56,22 @@ class DriverWarning(Warning):
         super(DriverWarning, self).__init__('Warning {0} occurred.\n\n{1}'.format(code, description))
 
 
+% if grpc_supported:
+class RpcError(Error):
+    '''An error specific to gRPC-based sessions'''
```

Comment:
Is "gRPC-based sessions" the way we refer to these remote sessions?
I want to make sure we're 100% consistent and I see a lot of names thrown around to refer to these:

remote sessions
proxy sessions
gRPC sessions
gRPC client sessions
gRPC-based sessions

All this paraphrasing only bring ambiguity for users.

Reply:
We have a class named `GrpcSessionOptions` and the comment on its use as a parameter says "MeasurementLink gRPC session options". @DavidCurtiss I think if you make the comment on your error class be "An error specific to gRPC sessions" we would at least be consistent. Then we can wordsmith it more later if someone really wants.

FWIW I don't see the term "gRPC client sessions" (or for that matter 'remote sessions' or 'proxy sessions') anywhere in nimi-python source, using `git grep`. Maybe those terms are prevalent in DVL efforts.

Reply:
"gRPC session" works

Reply:
Done